### PR TITLE
Adding Suspend and Resume APIs

### PR DESCRIPTION
**Pull request by athipp**

### CODE REVIEW

File: arm-asazure/2016-05-16/swagger/asazure.json [hunk 1]
```diff
@@ -192,6 +192,78 @@
...
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
```
Comment (fearthecowboy) [inline arm-asazure/2016-05-16/swagger/asazure.json]: You use `202 Accepted` -- is this supposed to be a long-running operation ( ie, use `"x-ms-long-running-operation": true` ?)

File: arm-asazure/2016-05-16/swagger/asazure.json [hunk 2]
```diff
@@ -192,6 +192,78 @@
...
```
Comment (fearthecowboy) [inline arm-asazure/2016-05-16/swagger/asazure.json]: You use `202 Accepted` -- is this supposed to be a long-running operation ( ie, use `"x-ms-long-running-operation": true` ?)

**Review by fearthecowboy — CHANGES_REQUESTED** [review]
Looks pretty good, just questions about long-running operations.

File: arm-asazure/2016-05-16/swagger/asazure.json (hunk 2 above)
Comment (athipp) [inline arm-asazure/2016-05-16/swagger/asazure.json]: good catch, added the property

File: arm-asazure/2016-05-16/swagger/asazure.json (hunk 1 above)
Comment (athipp) [inline arm-asazure/2016-05-16/swagger/asazure.json]: good catch, added the property

**Review by fearthecowboy — APPROVED** [review]

**Pull request merged**